addons/sale_stock/models/sale_order.py

```diff
@@ -35,9 +35,9 @@ class SaleOrder(models.Model):
         ('partial', 'Partially Delivered'),
         ('full', 'Fully Delivered'),
     ], string='Delivery Status', compute='_compute_delivery_status', store=True,
-       help="Red: Late\n\
-            Orange: To process today\n\
-            Green: On time")
+       help="Blue: Not Delivered/Started\n\
+            Orange: Partially Delivered\n\
+            Green: Fully Delivered")
```

Comment:
this probably needs to be added (at the right place) in `addons/sale_stock/i18n/sale_stock.pot`, so it can be translated.